dev-docs/RFCs/v1.0/tentative-feature.md

```diff
@@ -0,0 +1,59 @@
+# Tentative Features
```

Comment:
Nit: Look at deck RFCs, they have author, date etc here in the header. In fact, recommend you give the [RFC GUIDELINES](https://github.com/uber/deck.gl/blob/master/dev-docs/RFC-GUIDELINES.md) a read. Contains lesses from writing a bunch of these. Obviously not binding, Nebula can go its own way.

Reply:
Let's follow the same guidelines.